src/app/commands/cmd_options.cpp

```diff
@@ -926,6 +926,18 @@ class OptionsWindow : public app::gen::Options {
       }
     }
 
+    // Get the extension information from the compressed
+    // package.json file.
+    const ExtensionInfo info =
+      App::instance()->extensions().getCompressedExtensionInfo(filename);
+    // Check if the filename corresponds to aseprite-default theme
+    if (base::string_to_lower(info.name) ==
+        Extensions::kAsepriteDefaultThemeName) {
+      ui::Alert::show(
+        fmt::format(Strings::alerts_cannot_install_default_extension()));
```

Comment:
No need for `fmt::format()`:
```suggestion
        Strings::alerts_cannot_install_default_extension());
```